src/app/SIPUserAgents/SIPRegistrationUserAgent.cs

```diff
@@ -104,7 +104,10 @@ public bool IsRegistered
             string username,
             string password,
             string server,
-            int expiry)
+            int expiry,
+            int maxRegistrationAttemptTimeout = 60,

```

Comment:
No problem with adding these parameters with default values but could you leave the default values as constants. Avoids magic numbers.